pkg_defs/aca_view/build.sh

```diff
@@ -0,0 +1 @@
+pip install --no-deps --verbose --no-binary :all: --no-index .
```

Comment:
And this could also be python setup.py install etc but if already built this is also fine.  Not sure where we should take another action to do more recipe cleanup or if we'll just do that for shiny?

Reply:
Right. I guess these changes can be made for shiny (that was my assumption when John asked about the acis packages the other PR). Either that, or we create an issue about it now.